backend/docs/api/management_v2_deployments.yaml

```diff
@@ -0,0 +1,1286 @@
+                    rootfs-image.checksum: 32714818ad6f98ee0185a52e23a475d89122e3efd2b2c26c733781c28e798c99
+                    rootfs-image.version: test
+                  artifact_depends:
+                    device_type:
+                    - test
+                  clears_artifact_provides:
+                  - rootfs-image.*
+                  size: 36891648
+                  modified: 2016-03-11T13:03:17.063493443Z
+                - id: 0c13a0e6-6b63-475d-8260-ee42a590e8ff
+                  name: Application 1.0.0
+                  description: Johns Monday test build
+                  device_types_compatible:
+                  - Raspberry Pi
+                  info:
+                    format: mender
+                    version: 3
+                  signed: false
+                  updates:
+                  - type_info:
+                      type: rootfs-image
+                    files:
+                    - name: rootfs-image-1
+                      checksum: cc436f982bc60a8255fe1926a450db5f195a19ad
+                      size: 123
+                      date: 2016-03-11T13:03:17.063+0000
+                    meta_data: []
+                  artifact_provides:
+                    artifact_name: test
+                    rootfs-image.checksum: 32714818ad6f98ee0185a52e23a475d89122e3efd2b2c26c733781c28e798c99
+                    rootfs-image.version: test
+                  artifact_depends:
+                    device_type:
+                    - test
+                  clears_artifact_provides:
+                  - rootfs-image.*
+                  size: 36891648
+                  modified: 2016-03-11T13:03:17.063493443Z
+              - name: my-app-v2.0.0
+                artifacts:
+                - id: 0c13a0e6-6b63-475d-8260-ee42a590e8ff
+                  name: Application 2.0.0
+                  description: Johns Monday test build
+                  device_types_compatible:
+                  - Beagle Bone
+                  info:
+                    format: mender
+                    version: 3
+                  signed: false
+                  updates:
+                  - type_info:
+                      type: rootfs-image
+                    files:
+                    - name: rootfs-image-1
+                      checksum: cc436f982bc60a8255fe1926a450db5f195a19ad
+                      size: 123
+                      date: 2016-03-11T13:03:17.063+0000
+                    meta_data: []
+                  artifact_provides:
+                    artifact_name: test
+                    rootfs-image.checksum: 32714818ad6f98ee0185a52e23a475d89122e3efd2b2c26c733781c28e798c99
+                    rootfs-image.version: test
+                  artifact_depends:
+                    device_type:
+                    - test
+                  clears_artifact_provides:
+                  - rootfs-image.*
+                  size: 36891648
+                  modified: 2016-03-11T13:03:17.063493443Z
```

Comment:
could you double check that this is formatted correctly, I maybe seeing things.
```yaml
                  clears_artifact_provides:
                  - rootfs-image.*
                  size: 36891648
                  modified: 2016-03-11T13:03:17.063493443Z
```
looks strange/scary.

Reply:
`clears_artifact_provides` is an array of strings it's correct, although I would prefer an indentation, the generator didn't produce it and I don't want to create unnecessary conflicts with enterprise.